test/extensions/transport_sockets/tls/utility_test.cc

```diff
@@ -29,12 +29,28 @@ TEST(UtilityTest, TestDnsNameMatching) {
   EXPECT_TRUE(Utility::dnsNameMatch("lyft.com", "lyft.com"));
   EXPECT_TRUE(Utility::dnsNameMatch("a.lyft.com", "*.lyft.com"));
   EXPECT_TRUE(Utility::dnsNameMatch("a.LYFT.com", "*.lyft.COM"));
+  EXPECT_TRUE(Utility::dnsNameMatch(".lyft.com", "*.lyft.com"));
```

Comment:
Is this case expected and/or valid?

Reply:
True, this case is not expected or not valid. I had only included for test coverage reason. I removed now since a certificate of this SAN is not possible and probably will be treated as `lyft.com`